css/css-masking/parsing/resources/parsing-testcommon.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+function test_valid_value(property, value, serializedValue) {
```

Comment:
There is nothing like that in testharness already? Wonder if that should e considered adding. After all there are many specs which define CSS properties.

Reply:
I think testharness shouldn't be CSS specific. It might be good to have a css/resources directory that all the CSS specs can access.

This parsing-testcommon.js can easily move to such a directory if/when it is created.